build.cake

```diff
@@ -359,7 +359,7 @@ Task("CreateLinuxPackages")
             $"{Environment.CurrentDirectory}/artifacts/OctopusTools.{nugetVersion}.linux-x64.extracted:/app",
             $"{Environment.CurrentDirectory}/artifacts:/out"
         }
-    }, "octopusdeploy/bionic-fpm:latest", "bash /build/create-linux-packages.sh");
+    }, "octopusdeploy/package-linux-docker:latest", "bash /build/create-linux-packages.sh");
```

Comment:
Right, I see this is in line with this corresponding rename in the project that builds these images: https://build.octopushq.com/admin/settingsDiffView.html?id=buildType:Infrastructure_DockerImages_LinuxPackagerDebRpm&versionBefore=9&versionAfter=10&actionId=475767